src/Infrastructure/RedisCustomerRepository.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace WorkSpace\PersonService\Infrastructure;
+namespace Infrastructure;
```

Comment:
Your name spacing looks off. I would recommend you go with something like:

Nunchuck\SmartMeals

as the root, meaning that this would be

Nunchuck\SmartMeals\Infrastructure